Assets/VRTK/SDK/Daydream/DaydreamReach.cs

```diff
@@ -0,0 +1,87 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// DaydreamReach component uses touchpad to extend controller position out in front of user
+/// along the current controller orientation plane
+/// </summary>
+public class DaydreamReach : MonoBehaviour
+{
+    // controller to track, defaults to ./Controller but probably want a Pointer Joint like GvrControllerPointer/Laser
+    public Transform controller;
+
+    // maximum reach distance from controller origin
+    public float reachDistance = 0.5f;
+
+    private Vector3 positionOrigin;
+
+    void Start()
+    {
+        if (controller == null)
+        {
+            controller = transform.FindChild("Controller");
+        }
+        positionOrigin = transform.position;
+    }
+
+    void Update()
+    {
+        if (!GvrController.IsTouching)
+        {
+            // snap back to origin
+            transform.position = positionOrigin;
+        }
+        else
+        {
+            transform.position = ReachForwardByDistance();
+        }
+    }
+
+    private Vector3 ReachForwardByDistance()
+    {
+        Vector3 pos = positionOrigin;
+        Vector3 offset = Vector3.zero;
+        Vector2 touch = GetTouch(5);
+        //Debug.Log("touch: " + touch);
```

Comment:
If the line isn't required then just remove it. better to have cleanliness than commented out code :)